### PR TITLE
BZ#1903408: lift ETP limitation from OVN-K

### PR DESCRIPTION
**Pull request by mikemckiernan**
Fixes the doc angle for https://bugzilla.redhat.com/show_bug.cgi?id=1903408.

----
Notice that the limitation for external traffic policy is no longer present in the [OVN-Kubernetes limitations](https://deploy-preview-40824--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-limitations_about-ovn-kubernetes).

For 4.9, the [limitation](https://docs.openshift.com/container-platform/4.9/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-limitations_about-ovn-kubernetes) (first bullet) is present.

### CODE REVIEW

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 1]
```diff
@@ -5,11 +5,7 @@
 [id="nw-ovn-kubernetes-limitations_{context}"]
 = OVN-Kubernetes limitations
 
-The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
-
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
```
Comment (tssurya) [inline modules/nw-ovn-kuberentes-limitations.adoc:10]: @mikemckiernan: We still do not support Internal Traffic Policy. So please keep this bullet point intact but remove "external traffic policy" from it.

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 2]
```diff
@@ -5,11 +5,7 @@
...
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
```
Comment (tssurya) [inline modules/nw-ovn-kuberentes-limitations.adoc:12]: We should refactor this part to say this limitation can affect you when you add a service of type `ClusterIP`, `LoadBalancer`, `NodePort` or add a service with externalIP, since ITP can be done on a clusterIP as well.

**Review by tssurya — COMMENTED** [review]
thanks for the quick pr!

File: modules/nw-ovn-kuberentes-limitations.adoc (hunk 2 above)
Comment (mikemckiernan) [reply modules/nw-ovn-kuberentes-limitations.adoc:12]: Wow, thank you @tssurya.  I revised to bring the bullet back, but remote ETP and ack that ClusterIP is affected too.

File: modules/nw-ovn-kuberentes-limitations.adoc [hunk 3]
```diff
@@ -7,9 +7,8 @@
 
 The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
 
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
-The default value, `cluster`, is supported for both parameters.
-This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
+* OVN-Kubernetes does not support setting the internal traffic policy for a Kubernetes service to `local`.
+This limitation can affect you when you add a service of type `ClusterIP`, `LoadBalancer`, `NodePort`, or add a service with an external IP.
```
Comment (nalhadef) [inline modules/nw-ovn-kuberentes-limitations.adoc:11]: I'd reword this to avoid "you," since the process is being impacted, not the user. A minor nit.
